commands/drinksCommands.ts

```diff
@@ -0,0 +1,274 @@
+import { Client, Collector, Message } from 'discord.js'
+import { AbstractCommands } from '../Abstracts/AbstractCommand'
+import { ICommandElement } from '../General/commands'
+import { MessageHelper } from '../helpers/messageHelper'
+import { CardCommands } from './cardCommands'
+import { globals } from '../globals'
+
+export class DrinksCommands extends AbstractCommands {
+    private playerMap: Map<string, number>
+    private playerMapReversed: Map<number, string>
+    private playerCard: Map<number, string>
```

Comment:
Trengs det ein egen variabel for dette? Kan det ikkje ver ein del av playerMap?

playerMap: Map<string, IUserObject >

Der IUserObject e någe ala
```
interface IUserObject {
id: number;
card: string;
}
```